.github/workflows/pr-e2e-tests.yaml

```diff
@@ -1,6 +1,10 @@
 name: 'Neo4j-GenAI PR E2E Tests'
 
 on:
+  push:
+    branches:
+      - main
+      - fix-out-of-space-error-on-gh-actions
```

Comment:
I will remove this after the review. This is to test that this workflow works when I push to my working branch